src/components/NavBar/NavBar.tsx

```diff
@@ -60,13 +60,14 @@ const DropdownMenu = (props: any) => {
         <DropdownItem>Home</DropdownItem>
       </Link>
 
-      <Link
+      {!userReport &&<Link
```

Comment:
The journey only happens when there is no report now, so the user can only have one report.